requirements.txt

```diff
@@ -1,6 +1,6 @@
-charmhelpers~=1.1.0
-ops~=1.3.0
+charmhelpers @ git+https://github.com/juju/charm-helpers@master
 click
-fabric~=2.6.0
+fabric
+ops
 python-openstackclient
-PyYAML~=6.0
\ No newline at end of file
+PyYAML
```

Comment:
@aieri does this affect the charm build for bionic or focal though?

Reply:
possibly, but we could drop both